encoder/vaapiencoder_base.cpp

```diff
@@ -89,12 +89,12 @@ YamiStatus VaapiEncoderBase::start(void)
 void VaapiEncoderBase::flush(void)
 {
     AutoLock l(m_lock);
```

Comment:
@chivakker , "This patch was merged with obvious problems especially on the encoder base class. flush on the base class is doing nothing now. So the proper change would be to remove all calls to base::flush in all codecs. Please fix it."   Maybe we can fix h264 and h265 encoder in this patch series， and send a separated patch to fix other codecs?   We just to fix h264 and h265 b frame issue here,  what do you think?